html/elements/col.json

```diff
@@ -71,11 +71,11 @@
               },
               "firefox": {
                 "version_added": false,
-                "notes": "See bug <a href='https://bugzil.la/915'>915</a>."
+                "notes": "See bug <a href='https://bugzil.la/915'>bug 915</a>."
```

Comment:
Here too bug has been doubled up (i.e., "See bug bug 915"). This is true for all the edits on this file (`col.json`)